src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -0,0 +1,221 @@
+                                    isUsingDefaultAvatar={!this.state.previewAvatarURL}
+                                    onImageSelected={this.uploadAvatar}
+                                    onImageRemoved={this.removeAvatar}
+                                />
+
+                                <ExpensiTextInput
+                                    label={this.props.translate('workspace.editor.nameInputLabel')}
+                                    containerStyles={[styles.mt4]}
+                                    onChangeText={name => this.setState({name})}
+                                    value={this.state.name}
+                                    hasError={!this.state.name.trim().length}
+                                    errorText={this.state.name.trim().length ? '' : this.props.translate('workspace.editor.nameIsRequiredError')}
+                                />
+
+                                <ExpensiPicker
+                                    label={this.props.translate('workspace.editor.currencyInputLabel')}
+                                    onChange={currency => this.setState({currency})}
+                                    items={this.getCurrencyItems()}
+                                    value={this.state.currency}
+                                    style={[styles.mt4]}
+                                    isDisabled={hasVBA}
+                                />
+                                <Text style={[styles.textLabel, styles.colorMuted, styles.mt2]}>
+                                    {this.props.translate('workspace.editor.currencyInputHelpText')}
+                                </Text>
+                            </View>
+
+                            <FixedFooter style={[styles.w100]}>
+                                <Button
+                                    success
+                                    isLoading={policy.isPolicyUpdating}
+                                    text={this.props.translate('workspace.editor.save')}
+                                    onPress={this.submit}
+                                    pressOnEnter
+                                />
+                            </FixedFooter>
+                        </View>
+                    </>
+                )}
+            </WorkspacePageWithSections>
+        );
+    }
+}
+
+WorkspaceSettingsPage.propTypes = propTypes;
+WorkspaceSettingsPage.defaultProps = defaultProps;
+
+export default compose(
+    withOnyx({
+        betas: {
+            key: ONYXKEYS.BETAS,
+        },
+        policy: {
+            key: (props) => {
+                const routes = lodashGet(props.navigation.getState(), 'routes', []);
+                const routeWithPolicyIDParam = _.find(routes, route => route.params && route.params.policyID);
+                const policyID = lodashGet(routeWithPolicyIDParam, ['params', 'policyID']);
+                return `${ONYXKEYS.COLLECTION.POLICY}${policyID}`;
```

Comment:
This fancy logic should be necessary anymore because the `WorkspaceSettingsPage` has `props.route.params.policyID`. We _needed_ to do this in the `WorkspaceSidebar` since it was "drawerContent" before and not an actual screen.